ansible/host_vars/.gitignore

```diff
@@ -1 +0,0 @@
-localhost
```

Comment:
Why remove this file?
Especially since our docs still reference it: http://iplantcollaborativeopensource.github.io/DE/ansible/work_instructions/local_configs.html (step 3).